recipes/async_generator/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "async_generator" %}
+{% set version = "1.9" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "b7d5465c6174fe86dba498ececb175f93a6097ffb7cc91946405e1f05b848371" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - async_generator
+    - async_generator._tests
+
+about:
+  home: https://github.com/python-trio/async_generator
+  license: MIT
+  license_family: MIT
+  license_file: ''
+  summary: Async generators and context managers for Python 3.5+
+  description: ".. image:: https://img.shields.io/badge/chat-join%20now-blue.svg\n   :target: https://gitter.im/python-trio/general\n   :alt: Join chatroom\n\n.. image:: https://img.shields.io/badge/docs-read%20now-blue.svg\n\
```

Comment:
This is unreadable. I would just take a paragraph or 2 and avoid the examples

Reply:
Fixed, thanks